### PR TITLE
Custom insert popover problem Solved 

### LINKED ISSUE #750: Multiple pop-ups at once when tool buttons are clicked.
Screenshot 2021-10-13 at 12 30 05 AM

I clicked on several tool buttons such (e.g table, link etc...) and all corresponding popups occurs. However, they stay on screen until you click again on the icon that triggered the pop-up.

I expect a pop-up to close when I do one of the following:
* click the `esc` button,
* click on the page,
* click on another pop-up (to close previously opened pop-up).

This issue can be replicated on the [publiclab editor](https://publiclab.org/post) by clicking the tool buttons as you see in the image above.

### PR DESCRIPTION
**Pull request by RaviAnand111**
fixes #750 
fixes #793 

This is a PR for the multiple popover problem, all the popover buttons are being solved and committed in different PRs, as I have been told, every different changes should be made in different PRs, 
You can review the code 
@NARUDESIGNS @TildaDares 
Thank You

### CODE REVIEW

File: dist/PublicLab.Editor.js [hunk 1]
```diff
@@ -22804,6 +22807,21 @@ module.exports = function CustomInsert(_module, wysiwyg) {
       }
     });
   });
+
+  $(':not(".woofmark-command-insert")').click((e) => {
+    // check to see that the clicked element isn't fa-insert icon, else when you click on the fa-insert icon, the popover will close
+    try {
+      if ($('.woofmark-command-insert').children()[0] != e.target) {
```
Comment (NARUDESIGNS) [inline dist/PublicLab.Editor.js:22814]: I think you should also check if the popover is open here by adding 

```diff
+  if (popoverIsOpen && $('.woofmark-command-insert').children()[0] != e.target) {
```

this way, you don't need a try/catch block @RaviAnand111

Comment (RaviAnand111) [reply dist/PublicLab.Editor.js:22814]: Oh, Ok I will be adding that in the code, Thanks

**Comment by RaviAnand111** [comment]
Yeah, @NARUDESIGNS , Its late here, I have to go to sleep, I will be making those demo videos in the morning and adding it in this PR and the preivous one too, so that you and @TildaDares can review it. 
Thanks alot for helping

**Comment by RaviAnand111** [comment]
https://user-images.githubusercontent.com/59113935/150097299-2769ce73-dccd-4748-9bf6-fa16faf90ca3.mp4

Hey @NARUDESIGNS @TildaDares @jywarren , I have added a demo video, you can check out the changes before and after adding the code. 
ThankYou

**Pull request closed without merging**